polaris-core/src/main/resources/schemas/policies/system/data-compaction/2025-02-03.json

```diff
@@ -0,0 +1,41 @@
+{
+  "license": "Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)",
+  "$id": "https://polaris.apache.org/schemas/policies/system/data-compaction/2025-02-03.json",
+  "title": "Data Compaction Policy",
+  "description": "Inheritable Polaris policy schema for Iceberg table data compaction.",
+  "type": "object",
+  "properties": {
+    "version": {
+      "type": "string",
+      "const": "2025-02-03",
+      "description": "Schema version."
+    },
+    "enable": {
+      "type": "boolean",
+      "description": "Enable or disable data compaction."
+    },
+    "target_file_size_bytes": {
+      "type": "number",
+      "description": "Target data file size in bytes."
+    },
+    "config": {
+      "type": "object",
+      "description": "A map containing custom configuration properties. Please note that interoperability is not guaranteed.",
+      "additionalProperties": {}
+    }
+  },
+  "required": ["enable"],
```

Comment:
The schema version is optional. Polaris will pick up the latest version as the default version.